h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -139,7 +148,9 @@ public static Leaderboard getOrMakeLeaderboard(String project_name, UserFeedback
       return exists;
     }
 
-    return new Leaderboard(project_name, userFeedback, leaderboardFrame);
+    Leaderboard newLeaderboard = new Leaderboard(project_name, userFeedback, leaderboardFrame);
+    DKV.put(newLeaderboard);
+    return newLeaderboard;
```

Comment:
Why not `return new Leaderboard()` like before?

Reply:
Because we need to sometimes create new Leaderboards without put()ing them.  It was bad practice (a shortcut) for me to put the `DKV.put()` inside the constructor in the first place.  I moved it out, and now the clients must call `DKV.put()` if they want the Leaderboard to be in the DKV.